### PR TITLE
Initial setup of the project

### PR DESCRIPTION
**Pull request by Untouchamp**
Webpack TS ESLint REACT, PRETTIER + PRE-COMMIT hook (husky)

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function App() {
+    const ss = 'text'
```
[the remark this answers is not in the document]
Comment (Untouchamp) [reply src/components/App.tsx:4]: Prettier is responsible for that and I managed it not to set semicolon, should I make it to ends lines with semi?

File: src/components/App.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function App() {
```
[the remark this answers is not in the document]
Comment (Untouchamp) [reply src/components/App.tsx]: Done @Gudashev

File: package.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.tsx",
+  "repository": "https://github.com/Untouchamp/todo-list-mg.git",
+  "author": "David Kuchukhidze <untouchamp@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open --hot",
+    "build": "webpack --mode production",
+    "test": "react-scripts test",
```
[the remark this answers is not in the document]
Comment (Untouchamp) [reply package.json]: Done @Gudashev

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    env: {
+        browser: true,
+        node: true,
+        es2021: true,
+    },
+    extends: [
+        // 'eslint:recommended',
```
Comment (Gudashev) [inline .eslintrc.js:8]: Try to avoid commented code

File: src/components/App.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gudashev) [reply src/components/App.tsx:4]: end lines with semi

**Review by Gudashev — APPROVED** [review]

File: .eslintrc.js (hunk 4 above)
Comment (Untouchamp) [reply .eslintrc.js]: Done

File: src/components/App.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Untouchamp) [reply src/components/App.tsx]: done

**Pull request merged**